tests/Doctrine/Tests/ORM/Cache/DefaultRegionTest.php

```diff
@@ -99,3 +96,52 @@ public function testGetMulti()
         $this->assertEquals($value2, $actual[1]);
     }
 }
+
+final class SharedArrayCache extends ArrayCache
```

Comment:
I can move it to its own file but I didn't want to have it available for other tests...